cypress/utils/autoScrollableMenuSelected.ts

```diff
@@ -9,11 +9,15 @@ import {
  * @param selectedLabelIndex The index of the label to be selected
  * @param {[{ label: String }]} labels The list of all labels
  */
-export const verifySelectedMenu = (selectedLabelIndex, labels) => {
-  labels.forEach(({ id }, index) => {
+// TODO: update this code to really check if correctly selected
```

Comment:
todo

Reply:
I have created an new issue for that if it is ok.